hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/spi/impl/StorageContainerServiceProviderImpl.java

```diff
@@ -179,4 +185,38 @@ public DBCheckpoint getSCMDBSnapshot() {
     }
     return null;
   }
+
+  public DBCheckpoint getSCMDBSnapshotHA() throws IOException {
```

Comment:
Refactor this new method and `getSCMDBSnapshot()` to reduce code duplication.

Reply:
Sure will do that.